### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by kintaroju** [rzander/sccmclictr#9]
Hi Roger,

You have a great tool, and most of the functions are pretty self explanatory, but just was wondering if you have any plans to release like a basic document on some of the other functions eg stuff on your "Maintenance" icon on the Agent Actions Ribbon?

Cheers

**Comment by SCCMAdmin** [comment]
Hi @kintaroju,

In response to the following:  "Is any part of client center documented? I’m going to start to document this to be able to train the lower levels of support" (i.e. L1 and L2, as I’m like L3) ... what I received from Roger was the following:
"There is currently no Wiki on GitHub, please check [http://sccmclictr.codeplex.com/documentation](url) for any documentation."

I found the documentation back at codeplex useful. Nice work actually. I feel I’ve seen it years ago and it has given me a few more ideas again (re customisations).  Re documentation what I was thinking of was not admin docs re installation, configuration and deployment which is what we have now and I love them. 

I was more so thinking of an actual walk through of the product so L1 and L2 staff with no SCCM knowledge can safely use the ever growing tool without causing damage to agents, software installations* and ultimately less interruption to customers / end- users.
*(some packages should be re-run and some shouldn't, best to move to applications to remove this challenge but we have 1000s so not easy)

Maybe there can be some simple collaboration here on splitting up the documentation work and sharing what we've done. I would be open to it. I'm sure everyone that is involved would have time saved. :-)

Cheers!

**Comment by kintaroju** [comment]
Hi @SCCMAdmin ,

I echo the sentiments on the documentation which is more so software installs, and all the stuff in the ribbon as it isn't documented (although some of it is pretty apparent and straightforward) and would be great to put some documentation on using the application. 

Installation documents are straightforward, which isn't the issue and the stuff on codeplex is correct. 

If a Wiki is placed together on Github I'd be game for it to add some content as it'll probably help others as well

Cheers!

**Comment by rzander** [comment]
I've just created an empty wiki page... feel free to contribute some content :-)
There is also a very old one for the old SMS Client Center : https://netcologne.dl.sourceforge.net/project/smsclictr/Documentation/English%20V1.0/SMS_Client_Spy_EN_V1.0.pdf

**Comment by kintaroju** [comment]
Hi @rzander ,

Doesn't seem there is any general access to edit the Wiki, when you have a moment can you edit the Wiki permissions?

Cheers!

**Comment by rzander** [comment]
Oops, fixed now ?

**Comment by kintaroju** [comment]
Works like a charm now :D.

**Comment by kintaroju** [comment]
Hi @rzander ,

I was editing the wiki slowly, and noticed that Github doesn't have a option to store images natively into the wiki, the only way would be to link them:

https://help.github.com/articles/adding-images-to-wikis/

So not sure where you want the images stored?

Cheers!

**Comment by rzander** [comment]
You can add/attach the Images on this Issue... and then on wiki use a link (URL) to the image . Even if the issue is closed, the links and the Images do still exist...

**Comment by kintaroju** [comment]
awesome, that is great to know, well if you find that I or anyone else are spamming the discussion thread here, you'll know why :D.

**Comment by SCCMAdmin** [comment]
@rzander do you mind adding the help wanted label to this issue please? it would be good for tracking and sharing the load.

I was a little confused by your client spy document link being shown as the old SCCM Client Center document. Two different things I think. Feel free to explain to me. ;-]

**Comment by rzander** [comment]
Agree, two different Tools, but a lot of functions are identical. The old Clent Center was using WMI and the new one is using PowerShell remoting... but at the end they will trigger the same Code from the Agent... I just posted the link if someone wants to copy some sections ;-)

**Comment by SCCMAdmin** [comment]
@kintaroju @rzander I've updated the Installation wiki page. What do you think?

I want to ask about this text... 
Check out: "Installation and Configuration for Windows Remote Management" for more information.
Where is this and what is this? is it supposed to be a link? It was text only before I edited. I think it needs improvement / correction.

The current page, Installation, should be renamed. When I see Installation, I expect to see information about installing sccmclictr. Command line stuff, deployment info, other tricks and tips for things as simple as one installation to 100s of deployed installations. I will work to bring over the items from codeplex @rzander pointed me to recently. 

I suggest the current page be renamed to Pre-requisites and Required Configuration. Thoughts please. 

Cheers.

**Comment by rzander** [comment]
Started with some Installation Options... Feel free to comment or ask questions as there may things that are clear to me but not for everyone...

**Comment by kintaroju** [comment]
Hi @rzander ,

I'll be posting some pics/docs in the very near future, so you'll see this thread spammed with pictures.

Cheers!

**Comment by rzander** [comment]
go on ;-)

**Comment by SCCMAdmin** [comment]
Hey Gang, I'm about to embark on a command line customised installation effort, so we'll see how far I will get with Roger's current instructions. I'd like to share my installation example as a documentation example of a command line installation customised for use in an enterprise and delivered by SCCM. That's the goal. I'm sure others can learn from this. :-)

In the meantime I hope you've all been well.

@rzander a question for you, do you know if latest CC2012 works normally with SCCM Current Branch (CB)?
My assumption is yes as WinRM connection method hasn't changed and the SCCM client version is only really advancing forward in version but most functions will be the same. Would like your confirmation at some point and if you don't have SCCM CB anyone else is free to answer if you use SCCM CB already. Thanks.

**Comment by SCCMAdmin** [comment]
@rzander would you please make this PDF available for download in GitHub please ... [https://www.codeplex.com/Download?ProjectName=sccmclictr&DownloadId=861795](url)
There are good tips within but none of them are available here. Also this document may need to be rewritten.

I can't get the MSI to work properly after it is extracted from the EXE. Will log new issues. There are multiple. If I use the EXE, downsides are the MSI transform needs a hardcoded path because it is not in the temporary installation dir. Likewise the external and customised SCCMCliCtrWPF.exe.config is not found and you need to dump it into C:\Users\xxx\AppData\Roaming\Zander Tools\Client Center for Configuration Manager 1.0.3.8\install manually. Can't use the MSI as it says it must be launched with bootstrapper. So stuck either way to continue enterprise command line installation as it needs manual interaction. Customisations are working fine once installed so part of the PDF doc info is a must.

Thanks.

**Comment by rzander** [comment]
latest Client Center does not have a "2012" in the Name because it does alswo work with ConfigMgr CB...

**Comment by rzander** [comment]
The current Setup is using the bootstrap .exe because it contains a x86 and x64 MSI. I wasn't aware that the extract function does not provide a usable msi... If this is the case I will think about to publish two MSI's instead of one .exe ?! Would that help ?...  on the other side, who is using the x86 Version ? None of my customers do have any x86 Systems...

**Comment by SCCMAdmin** [comment]
Thanks for confirming sccmclictr :-) or CCCM as I will now have to call it works with SCCM CB. I've called it CC2012 for so long I'm just excited about it. Hehe. We actually also still use sccmclictr2007 as I call it, So I use the years to differentiate in conversation. Yes, it's high time I drop the 2012!

I wonder if CCCM or CC4CM is the newer name I will use internally. Client Center for Configuration Manager is descriptive as it should be but a mouthful. Microsoft didn't help with System Center Configuration Manager Current Branch. Couldn't be any longer really. :-]

I definitely suggest start with making the 64-bit MSI available. You can work later to create a 32-bit MSI if requested, if not, save some work too.

**Comment by sbf-** [comment]
To install the MSI only (without bootstrapper), add SETUPEXEDIR=1 on the msiexec command line
It's case sensitive.

**Comment by rzander** [comment]
That was the Trick !!  THX for sharing...

**Comment by rzander** [comment]
Install-Wiki updated: https://github.com/rzander/sccmclictr/wiki/3)-Installation

**Comment by SCCMAdmin** [comment]
Thanks @sbf- and @rzander. I still haven't even got around to trying it. I'll let you know once I do as then a package I can create and deliver to L2 team members will be readied and I will need to document some items which will help with their use of CCCM and also this thread to update notes in the wiki.

**Comment by SCCMAdmin** [comment]
Yo Gang! I've added Optional Configuration sub-section under the Installation page's Customization section. It's useful to me and least the poster of Issue #60 and I'm sure it will be useful to many more. Cheers.
